pkg/minikube/bootstrapper/kubeadm/versions.go

```diff
@@ -242,7 +242,14 @@ var versionSpecificOpts = []VersionedExtraOption{
 
 	// System pods args
 	NewUnversionedOption(Kubelet, "pod-manifest-path", "/etc/kubernetes/manifests"),
-	NewUnversionedOption(Kubelet, "allow-privileged", "true"),
+	{
+		Option: util.ExtraOption{
+			Component: Kubelet,
+			Key:       "allow-privileged",
+			Value:     "true",
+		},
+		LessThanOrEqual: semver.MustParse("1.14.2"),
```

Comment:
Is this still needed in 1.14.3 ? Maybe this should be < 1.15 ?

Reply:
@afbjorklund I checked change logs of v1.15.0-beta.1. The deprecated Kubelet flag --allow-privileged has been removed since v1.15.0-alpha.3. So here should change to <= 1.15.0-alpha.3.
Sorry for my mistake, and thanks for your reminder.